src/pages/Auth/Login/Login.jsx

```diff
@@ -0,0 +1,54 @@
+import { Link } from "react-router-dom";
+import { Navbar } from "../../../components/Navbar/Navbar";
```

Comment:
Create an index.js in pages and components folder and export them there. 
Doing that will help in importing several components from one main path and also keeps the path length short